credential/exchange/model.go

```diff
@@ -122,6 +130,13 @@ type ClaimFormat struct {
 	LDPVP *LDPType `json:"ldp_vp,omitempty" validate:"omitempty,dive"`
 }
 
+func SupportedClaimFormats() map[string]bool {
```

Comment:
I think this should return `[]string` but it would be nice to maintain the type on this API and handle the conversion on the caller. So maybe have it return `[]LinkedDataFormat`?